src/index.js

```diff
@@ -5,6 +5,7 @@ export { B3DMLoader } from './three/loaders/B3DMLoader.js';
 export { PNTSLoader } from './three/loaders/PNTSLoader.js';
 export { I3DMLoader } from './three/loaders/I3DMLoader.js';
 export { CMPTLoader } from './three/loaders/CMPTLoader.js';
+export { SUBTREELoader } from './three/loaders/SUBTREELoader.js';
```

Comment:
There shouldn't be any reason to export this publicaly, I don't think - the only reason PNTS, I3DM etc loaders were exported was incase users wanted to to use these formats separately. With all those formats being deprecated, though, it's possible there's less of a reason to export those, as well, but we can wait on that.